lib/rules/no-unknown-property.js

```diff
@@ -108,7 +108,7 @@ const SVGDOM_ATTRIBUTE_NAMES = {
 
 const DOM_PROPERTY_NAMES = [
   // Standard
-  'acceptCharset', 'accessKey', 'allowFullScreen', 'allowTransparency', 'autoComplete', 'autoFocus', 'autoPlay',
+  'acceptCharset', 'accessKey', 'allowFullScreen', 'allowtransparency', 'autoComplete', 'autoFocus', 'autoPlay',
```

Comment:
Did react remove support for this property? I’m reasonably sure it was supported at the time this list was created. What version dropped support for it?

The lowercase one is just a generic custom property name; there’s no need to include it in the list at all.

Reply:
Looks like it was removed in facebook/react#10823, which was released in 16.1.0. It must have been missed when making the release notes.